### DISCUSSION TITLE
проблема с экспортом локации.

### DISCUSSION THREAD

**Discussion by naparn1k** [PavelBlend/blender-xray#657] [Q&A]
при импорте локации из сталкера в блендер все хорошо(есть текстуры), но экспорте данной локации в fbx и последующем импорте в юнити пропадает текстура. Как можно экспортировать в fbx , что бы не пропадал текстура?

**Comment by PavelBlend** [comment]
@naparn1k я не использую fbx и юнити, поэтому не знаю. Лучше спросить на форумах и группах по юнити.

**Comment by s30-zip** [comment]
> при импорте локации из сталкера в блендер все хорошо(есть текстуры), но экспорте данной локации в fbx и последующем импорте в юнити пропадает текстура. Как можно экспортировать в fbx , что бы не пропадал текстура?

https://youtu.be/fEuK6lKzAEI

**Comment by duppehnsk** [comment]
> при импорте локации из сталкера в блендер все хорошо(есть текстуры), но экспорте данной локации в fbx и последующем импорте в юнити пропадает текстура. Как можно экспортировать в fbx , что бы не пропадал текстура?
> 
> никак) а давным давно писал про данную проблему, мне сказали что это проблема с модулем fbx,что он путь текстуры конвертирует в название материала, вот и все проблемы. Лично я в Юнити просто написал парсер при старте уровня, он сам меняет материалы на нужные и сохраняет префаб на карте

**Reply by naparn1k** [reply]
Поделись парсером:)

**Reply by naparn1k** [reply]
Поделись парсером:)
